website/docs/getting-started/setup/instance-configuration/authentication/google-login.md

```diff
@@ -7,64 +7,63 @@ sidebar_position: 1
 
 # Google OAuth
 
-Google OAuth is one of the most widely used custom authentication integrations. Appsmith allows you to integrate with Google for authenticating users to your apps.
+Appsmith allows you to integrate with Google to authenticate users to your apps.
 
-![Click to expand](</img/Google_Login.png>)
+## Configure Google Cloud
 
-To enable Google Sign in, login to your [google cloud console](https://console.cloud.google.com/)
+To enable Google Sign in, log in to your [google cloud console](https://console.cloud.google.com/) and follow the steps below:
 
-**1. Navigate to the** [**OAuth consent section**](https://console.cloud.google.com/apis/credentials/consent) **under APIs & Services**
+1. Navigate to the [**OAuth consent screen**](https://console.cloud.google.com/apis/credentials/consent) under **APIs & Services**.
 
-![Click to expand](</img/Google_OAuth_Consent_1.png>)
+  ![Click to expand](</img/Google_OAuth_Consent_1.png>)
 
-**2. Configure the consent screen with the domain on which you want to host Appsmith**
+2. Configure the consent screen with the domain you want to host Appsmith.
 
-![Click to expand](</img/Google_OAuth_Consent.png>)
+  ![Click to expand](</img/Google_OAuth_Consent.png>)
 
-**3. Navigate to the** [**credentials section**](https://console.cloud.google.com/apis/credentials) **and create new OAuth Credentials**
+3. Navigate to the [**credentials**](https://console.cloud.google.com/apis/credentials) and create new OAuth Credentials.
 
-![Click to expand](</img/Google_OAuth_Creds.png>)
+  ![Click to expand](</img/Google_OAuth_Creds.png>)
 
-**4. Configure the OAuth Credentials for a web application**
+4. Select the application type as a Web application. Update the following fields:
 
-**Javascript Origins:** the domain on which you are hosting Appsmith with HTTPS **`(ex: https://app.appsmith.com)`**
+    1. **Javascript Origins:** the domain on which you are hosting Appsmith with HTTPS **`(ex: https://app.appsmith.com)`**
```

Comment:
It should be Authorized JavaScript origins: 

also don't think this is HTTPS. please clarify this 
<img width="432" alt="Screenshot 2023-03-09 at 19 44 35" src="https://user-images.githubusercontent.com/32227135/224051772-89a7b5b1-7044-424c-a6b6-73bc87782c6e.png">